src/panoramic/cli/field/client.py

```diff
@@ -0,0 +1,121 @@
+import logging
+from typing import Any, Dict, List, Optional
+from urllib.parse import urljoin
+
+from panoramic.auth import OAuth2Client
+from panoramic.cli.clients import VersionedClient
+from panoramic.cli.config.auth import get_client_id, get_client_secret
+from panoramic.cli.config.field import get_base_url
+
+logger = logging.getLogger(__name__)
+
+
+class Field:
+    def __init__(
+        self,
+        *,
+        slug: str,
+        group: str,
+        display_name: str,
+        data_type: str,
+        field_type: str,
+        description: Optional[str],
+        data_source: Optional[str],
+        calculation: Optional[str],
+        aggregation: Optional[str],
```

Comment:
This should be a dictionary: https://github.com/panoramichq/diesel-service/blob/taxonomy-upsert/federated/taxonomy/models.py#L47

Reply:
Also - are we missing `display_format`? https://github.com/panoramichq/diesel-service/blob/taxonomy-upsert/federated/taxonomy/models.py#L49

Reply:
Good catch, must've forgotten those fields.